collections/CHANGELOG.md

```diff
@@ -35,4 +35,5 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [#14351](https://github.com/cosmos/cosmos-sdk/pull/14351) Add keyset
 * [#14364](https://github.com/cosmos/cosmos-sdk/pull/14364) Add sequence
 * [#14468](https://github.com/cosmos/cosmos-sdk/pull/14468) Add Map.IterateRaw API.
-* [#14310](https://github.com/cosmos/cosmos-sdk/pull/14310) Add Pair keys 
\ No newline at end of file
+* [#14310](https://github.com/cosmos/cosmos-sdk/pull/14310) Add Pair keys 
+* [#14397](https://github.com/cosmos/cosmos-sdk/pull/14397) Add IndexedMap
```

Comment:
TBH these changelog entries are superfluous -- as a dev, they tell me nothing useful. 

Honestly, we can probably have a single collections CL entry that outlines all the additions and changes.

Reply:
yes, it makes sense to me. When we release the first "stable" version we can add a comprehensive list of the features in the changelog, and then apply the classic by PR changelog after the first release.